mlrun/projects/project.py

```diff
@@ -3929,6 +3950,58 @@ def list_runs(
             **kwargs,
         )
 
+    def list_completed_runs(
+        self,
+        name: Optional[str] = None,
+        uid: Optional[Union[str, list[str]]] = None,
+        project: Optional[str] = None,
+        sort: bool = True,
+        iter: bool = False,
+        start_time_from: datetime = None,
+        start_time_to: datetime = None,
+        **kwargs,
+    ) -> mlrun.lists.RunList:
+        """Retrieve a list of runs, filtered by various options.
+
+        The returned result is a `` (list of dict), use `.to_objects()` to convert it to a list of RunObjects,
+        `.show()` to view graphically in Jupyter, `.to_df()` to convert to a DataFrame, and `compare()` to
+        generate comparison table and PCP plot.
+
+        Example::
+
+            # return a list of runs matching the name and label and compare
+            runs = project.list_runs(name="download", labels="owner=admin")
```

Comment:
Docstring need some updates
```suggestion
            runs = project.list_completed_runs(name="download", labels="owner=admin")
```